library/src/scala/compiletime/ops/package.scala

```diff
@@ -0,0 +1,174 @@
+package scala.compiletime
+
+import scala.annotation.infix
+
+package object ops {
+  object any {
+    /** Equality comparison of two singleton types.
+     * ```scala
```

Comment:
No need to specify scala here: that's the default inside scaladoc.

Reply:
This doesn't seem to be true. In the screenshot below, `&&` has the `scala` annotation on the Markdown code block, and `||` does not. I've therefore left the annotations in.

![image](https://user-images.githubusercontent.com/990794/71993344-390d8980-3237-11ea-8933-98a1738ce82a.png)